examples/js_of_ocaml/todomvc.ml

```diff
@@ -0,0 +1,423 @@
+
```

Comment:
I think it's more readable with this separation line because it clearly separates the `open` directive from the modules. But i guess it's a matter of taste.